File: Api/DataLayer.Dados/Entities/Demanda.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataLayer.Dados.Entities
{
    public class Demanda
    {
        public int IdDemanda { get; set; }
        public string Titulo { get; set; }
        public string Descricao { get; set; }
        public string Anexo { get; set; }
        public DateTime DataCadastro { get; set; }
    }
}

File: Web/src/components/questions/demanda.ts
export class Demanda {
    IdDemanda: number;
    Titulo: string;
    Descricao: string;
    Anexo: string;

    constructor(id?: number, titulo?: string, descricao?:string, anexo?:string){
        this.IdDemanda = id;
        this.Titulo = titulo;
        this.Descricao = descricao;
        this.Anexo = anexo;
    }
}